tests/test_dataset.py

```diff
@@ -466,47 +465,6 @@ def test_dataset_current_records_index_filtering_accurate_records_yielded(
     ]
 
 
-@pytest.mark.freeze_time("2025-05-22 01:23:45.567890")
-def test_dataset_write_includes_minted_run_timestamp(tmp_path):
```

Comment:
This test no longer makes sense as `TIMDEXDataset.write()` no longer mints a timestamp if one is not provided. 

Either `run_timestamp` is provided explicitly to each `DatasetRecord` for writing, or it defaults to an ISO timestamp version of `run_date` for that record (which is required).